InferenceSystem/src/model_retrain_negative_samples.py

```diff
@@ -0,0 +1,46 @@
+
+import argparse
+import datetime
+import globals
+
+def main():
+    """
+    Example usage: 
+    python3 PrepareDataForPredictionExplorer.py --start_time "2020-07-25 19:15" --end_time "2020-07-25 20:15" --dataset_folder /Users/prakrutigogia/Documents/Microsoft/AlwaysBeLearning/MSHack/Round4
+    """
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--start_time', type=str, required=True, \
+        help='Start time in PST in following format 2020-07-25 19:15')
+    parser.add_argument('--end_time', type=str, required=True, \
+        help='End time in PST in following format 2020-07-25 20:15')
+    parser.add_argument('hydrophone', choices=list(globals.S3_STREAM_URLS.keys()), required=True, \
+        help='Hydrophone location')
+
+    args = parser.parse_args()
+
+    datetime_format = '%Y-%m-%d %H:%M:%S'
```

Comment:
In example usage and argument help lines you omit seconds (`"2020-07-25 19:15"`) but here you use seconds, this is going to fail at runtime. I suggest adding seconds to examples rather than slashing accuracy to minutes only.